README.md

```diff
@@ -2,7 +2,7 @@
 
 [tModLoader] dedicated server  
 
-Terraria server 1.4.0.2 with tModLoader v0.11.7.1
+Terraria server 1.4.0.2 with tModLoader v0.11.7.1 NOTE: This will not function until tmodloader is updated
```

Comment:
What does this note refer to? Is it related to releases on steam?

Reply:
So It means do not merge this yet. When I wrote it I did not realize tmodloader has not yet added support for the new terraria 1.4 update. So all these changes will break until tmodloader updates, which from what ive read may take a while.